packages/widget-chat/src/reducers/conversation.js

```diff
@@ -12,18 +18,31 @@ function filterActivity(activity) {
   return filteredActivities.indexOf(activity.verb) === -1;
 }
 
+function sortActivityByTime(activities) {
+  activities = _.uniqBy(activities, `id`);
+  return _.sortBy(activities, [`published`]);
+}
+
 
-export default function conversation(state = {
+export default function reduceConversation(state = {
   activities: [],
   id: null,
   participants: [],
   isFetching: false,
   isLoaded: false,
+  isLoadingHistoryUp: false,
+  isLoadingHistoryDown: false,
```

Comment:
Ahhh good catch. This only happens if someone links to a specific activity item, which will never happen in the widget... for now...